content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,194 @@
+  calendar to schedule a time for this activity.
+- If **Upload Document** is selected, a link to upload a document will be added directly to the
+  planned activity in the chatter.
+- If **Request Signature** is selected, a link will be added to the planned activity in the chatter
+  that will open a signature request pop-up.
+
+.. note::
+   The actions available to select on an activity type will vary depending on the applications that
+   are currently installed in the database.
+
+Default user
+~~~~~~~~~~~~
+
+To automatically assign this activity to a specific user when this activity type is scheduled,
+choose a name from the :guilabel:`Default User` drop-down. If this field is left blank, the activity
+will be assigned to the user who creates the activity.
+
+Default summary
+~~~~~~~~~~~~~~~
+
+To include any notes every time this activity type is created, enter them into the
+:guilabel:`Default Summary` field.
+
+.. note::
+   The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields will be
+   included when an activity is created. However, they can be altered before the activity is
+   scheduled or saved.
 
-Odoo helps you to plan a flow of activities by allowing you to set *Recommended Next Activities*.
+Next activity
+-------------
 
-.. image:: plan_activities/recommended_activities.png
+To automatically suggest or trigger a new activity after an activity has been marked complete, the
+*chaining type* must be set.
+
+Suggest next activity
+~~~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Chaining Type` field, select :guilabel:`Suggest Next Activity`. The field
+underneath will now be titled :guilabel:`Suggest`. Select any activities to recommend as follow-up
+tasks to this activity type.
+
+In the :guilabel:`Schedule` field, choose a default deadline for these activities. This can be
+altered before the activity is scheduled.
+
+Click :guilabel:`Save`.
+
+.. figure:: plan_activities/schedule-recommended-activity.png
    :align: center
-   :alt: View of an activity type form emphasizing the field recommended next activities for Odoo
-         Discuss
+   :alt: Schedule activity popup with emphasis on recommended activities.
+
+   If an activity has the chaining type set to suggest, and has activities listed in the Suggest
+   field, users are presented with recommendations for activities as next steps.
 
-Once the respective activity is completed, select *Done & Schedule Next* and next steps are
-suggested to you.
+Trigger next activity
+~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: plan_activities/schedule_recommended_activity.png
+Setting the *chaining type* to *trigger* immediately launches the next activity once the previous
+one is completed.
+
+In the :guilabel:`Chaining Type` field, select :guilabel:`Trigger Next Activity`. The field
+underneath will now be titled :guilabel:`Trigger`. Select the activity that should be launched once
+this activity is completed.
+
```

Comment:
The email template field is only visible when a model is set, meaning it will only be visible for activity types created through a specific application, not the general settings. Since this doc is supposed to be more general I wanted to keep that option out of it for now, because I was worried that it would cause more confusion.